### PR TITLE
Upgrade redukt libs

### PR DESCRIPTION
**Pull request by krysthianlessa**
## Description

This PR aims to upgrade redukt gradle and libs.

### CODE REVIEW

File: core/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<manifest package="com.github.raulccabreu.redukt">
+<manifest package="com.github.krysthianlessa.redukt">
```
Comment (Marinofull) [inline core/src/main/AndroidManifest.xml:1]:
```suggestion
<manifest package="com.github.oxeanbits.redukt">
```

**Review by Marinofull — CHANGES_REQUESTED** [review]

File: ui/src/main/java/com.github.raulccabreu.redukt.ui/ReactiveActivity.kt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package com.github.raulccabreu.redukt.ui
```
Comment (Marinofull) [inline ui/src/main/java/com.github.raulccabreu.redukt.ui/ReactiveActivity.kt:1]:
```suggestion
package com.github.oxeanbits.redukt.ui
```

here too?

File: ui/src/main/java/com.github.raulccabreu.redukt.ui/ReactiveActivity.kt [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 
-import android.support.v7.app.AppCompatActivity
+import androidx.appcompat.app.AppCompatActivity
 import com.github.raulccabreu.redukt.Redukt
 import com.github.raulccabreu.redukt.states.StateListener
```
Comment (Marinofull) [inline ui/src/main/java/com.github.raulccabreu.redukt.ui/ReactiveActivity.kt:5]:
```suggestion
import com.github.raulccabreu.redukt.Redukt
import com.github.raulccabreu.redukt.states.StateListener
```

??

File: ui/src/main/java/com.github.raulccabreu.redukt.ui/ReactiveFragment.kt [hunk 4]
```diff
@@ -1,6 +1,6 @@
 package com.github.raulccabreu.redukt.ui
```
Comment (Marinofull) [inline ui/src/main/java/com.github.raulccabreu.redukt.ui/ReactiveFragment.kt:1]: ?

**Review by Marinofull — CHANGES_REQUESTED** [review]
Do we need to change all the instances of raulccabreu or only just those you changed in your fork?

Note that even file names are like that

**Comment by krysthianlessa** [comment]
> Do we need to change all the instances of raulccabreu or only just those you changed in your fork?
> 
> Note that even file names are like that

These are package names from the android project. I don't know if they can be changed.

**Pull request closed without merging**